models/staging/jaffle_shop/src_jaffle_shop.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+
+sources:
+    - name: jaffle_shop
```

Comment:
a secret move here, though not really because it is in the docs, _but_ if you wanted to name your sources something different from the schema they are stored in your could:
```
sources:
  - name: jaffle_store
     description: A clone of a Redshift application database.
     database: raw
     schema: jaffle_shop
     ....
```

You could then reference this with {{ source('jaffle_store',....) }}